src/webportal/src/app/job-submission/utils/constants.jsx

```diff
@@ -74,3 +76,23 @@ export const PAI_ENV_VAR = [
     desc: 'Index of current task in current task role, starting from 0',
   },
 ];
+export const PROTOCOL_TOOLTIPS = {
+  taskRoleName: 'Name of the taskRole, string in ^[A-Za-z0-9\\-._~]+$ format.',
+  taskRoleContainerSize: (
+    <div>
+      <p>Resource required per container instance</p>
+      <p>CPU number and memory number will be auto scaled with GPU number by default</p>
+    </div>
+  ),
+  taskRoleCPU: 'CPU number, unit is CPU vcore',
+  taskRoleMemory: 'Memory number, unit is MB',
+  taskRoleGPU: 'GPU number, unit is GPU card',
+  taskRoleDocker: `Container's docker image`,
+  jobInformation: 'Job Information',
```

Comment:
I don't know what should be displayed here. Any suggestions?

Reply:
"Job Spec"?
"Click to view job spec details"

Reply:
Job Information's tooltip is changed to a `learn more` link